taskfile.yaml

```
@@ -9,6 +9,7 @@ vars:
 
 tasks:
   clean:
+    desc: "Removes project builds."
```

```suggestion
    desc: "Removes the project build directory."
```
For now let's be more explicit. If `clean` changes we can make the description more abstract later.